include/sound/sof/topology.h

```diff
@@ -45,6 +45,13 @@ enum sof_comp_type {
 	SOF_COMP_FILEWRITE = 10001,	/**< host test based file IO */
 };
 
+/* Scheduling domain, unset, Low Latency, or Data Processing */
+enum sof_comp_domain {
+	SOF_COMP_DOMAIN_UNSET = 0,
+	SOF_COMP_DOMAIN_LL,
+	SOF_COMP_DOMAIN_DP,
+};
```

Comment:
This is IPC4 only, I think it should be in ipc4-topology.h like SOF_IPC4_NODE_TYPE and friends?

Reply:
Oh yes, now that the struct snd_sof_dai_link comp_domain datamember is u32, it can be defined there, or even in ipc4-topology.c. Earlier it could not.